src/components/Signup/Signup.jsx

```diff
@@ -17,36 +33,55 @@ export default function Signup() {
                     <label className='popup__form-label'>
                         Email:
                         <input
+                            name='email'
+                            onChange={handleChange}
                             placeholder='Enter email'
                             className='popup__form-input'
                             type='email'
                             required
                         />
-                        <p className='popup__form-input_type_error'></p>
+                        <p className='popup__form-input_type_error'>
+                            {errors.email && "Please enter a valid email"}
+                        </p>
                     </label>
                     <label className='popup__form-label'>
                         Password:
                         <input
+                            name='password'
+                            onChange={handleChange}
                             placeholder='Enter password'
                             className='popup__form-input'
                             type='password'
                             required
                         />
-                        <p className='popup__form-input_type_error'></p>
+                        <p className='popup__form-input_type_error'>
+                            {errors.password}
+                        </p>
                     </label>
                     <label className='popup__form-label'>
                         Username:
                         <input
+                            name='name'
+                            onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components